js/pihole/gravity.js

```diff
@@ -0,0 +1,36 @@
+function eventsourcetest() {
+    var alInfo = $("#alInfo");
+    var alSuccess = $("#alSuccess");
+    var ta = document.getElementById("output");
+    var source = new EventSource("php/gravity.sh.php");
+
+    alInfo.show();
+    alSuccess.hide();
+
+    source.addEventListener("message", function(e) {
+        if(e.data.indexOf("Pi-hole blocking is Enabled") !== -1)
```

Comment:
Perhaps change to check index of "Pi-hole blocking is ", because they might have disabled their Pi-hole but still are updating it.